django/cantusdb_project/templates/base.html

```diff
@@ -338,11 +339,46 @@ <h5>Database Links</h5>
                         </ul>
                     </div>
                     <div class="col-md" id="login">
-                        <h5>Login for Contributors</h5>
                         {% if not request.user.is_anonymous %}
```

Comment:
This `if` statement seems to be repeated in line 379. How about converting them into an `if`, `else` pair?